tests/baselines/reference/docker/vscode.log

```diff
@@ -1,11 +1,26 @@
-Exit Code: 0
+Exit Code: 1
 Standard output:
 yarn run vX.X.X
 $ gulp compile --max_old_space_size=4095
 [XX:XX:XX] Node flags detected: --max_old_space_size=4095
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-Done in ?s.
+[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(13,3): Duplicate identifier 'PrepareCallHierarchy'.
+[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(14,3): Duplicate identifier 'ProvideCallHierarchyIncomingCalls'.
+[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(15,3): Duplicate identifier 'ProvideCallHierarchyOutgoingCalls'.
```

Comment:
@rbuckton `vscode` seems to break when run with our nightly; seems like the call hierarchy types are defined in multiple places in incompatible ways? This error is, if I'm not mistaken, in the `protocol.d.ts` that `vscode`'s install copies out of the installed copy of `typescript`.